### PR TITLE
Records owned

### PR DESCRIPTION
**Pull request by EPashkin**
Part of #103

WIP.

### CODE REVIEW

File: src/analysis/functions.rs [hunk 1]
```diff
@@ -19,7 +19,7 @@ pub struct Info {
     pub glib_name: String,
     pub kind: library::FunctionKind,
     pub comented: bool,
-    pub class_name: Result,
+    pub owner_name: Result,
```
Comment (gkoz) [inline src/analysis/functions.rs]: `owner` would probably be clearer as `receiver` (as in receiver of the method) or `type`.

Comment (EPashkin) [inline src/analysis/functions.rs]: `type` is better

**Comment by EPashkin** [comment]
Renamed `owner_xxx` to `type_xxx`

**Comment by EPashkin** [comment]
Ready for merge.
Generated files is compiling in gtk (after replacement `*mut` to `*const` in sys)
with 2 warning about unused `use glib::types;` and `use object::*;`,
so I plan change `codegen::general::uses` in next PR

**Comment by EPashkin** [comment]
Removed TODO

**Comment by gkoz** [comment]
Thanks!
@homu r+

**Pull request merged**